### ISSUE TITLE
Set patch URL for download | pull folder

### ISSUE THREAD

**Issue by Milor123** [git-ftp/git-ftp#398]
# Question 

### What I want do?

- I want that git-ftp, can download only the indicated folder with the file in... 

`/htdocs/wp-content/plugins/ >> I need * files of plugins/`

### What is the problem?

My config is: 

```
git config git-ftp.url ftp://ftp.serverxD.com:21/htdocs/wp-content/plugins/
```

but when I use **`git ftp down`** OR **`git ftp pull`**
**It _download all files and folder from `/` to end_**

How should I configure the **git-ftp** for download only the `/plugins/` folder?

```
| → echo "$SHELL"
/bin/bash

| → uname -a
Linux kali 4.12.0-kali2-amd64 #1 SMP Debian 4.12.12-2kali1 (2017-09-13) x86_64 GNU/Linux

| → git-ftp version
git-ftp version 1.4.1-UNRELEASED
```

**Comment by mkllnk** [comment]
Thanks for reporting. That definitely sounds like a bug.

**Comment by DerTanzschuh** [comment]
I run into this bug during my analysis here: https://github.com/git-ftp/git-ftp/issues/366#issuecomment-490266930

So I’ve fixed this accidentally in https://github.com/DerTanzschuh/git-ftp/commit/b75ca79e11cdfe852eebc21d51d2e0a6453b88c4

**Comment by DerTanzschuh** [comment]
fixed by #526
